spec/unit/puppet/provider/rabbitmq_policy/rabbitmqctl_spec.rb

```diff
@@ -7,11 +7,10 @@
       pattern: '.*',
       definition: {
         'ha-mode' => 'all'
-      },
-      provider: described_class.name
```

Comment:
wasn't 100% sure about this, but seems to work without it.

Reply:
Yeh, it does :)